Library/Tracking/Koala.swift

```diff
@@ -2028,7 +2031,12 @@ public final class Koala {
 
     self.logEventCallback?(event, props)
 
-    self.client.track(
```

Comment:
Currently we just send all events to both clients. We will whitelist before this point.